web/libs/ui/src/lib/popover/popover.tsx

```diff
@@ -0,0 +1,41 @@
+import type * as React from "react";
+import {
+  Popover as ShadPopover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverAnchor,
+} from "../../shad/components/ui/popover";
+import { cn } from "@humansignal/shad/utils";
+
+export interface PopoverProps extends React.ComponentProps<typeof ShadPopover> {
+  trigger: React.ReactNode;
+  children: React.ReactNode;
+  align?: "start" | "center" | "end";
+  sideOffset?: number;
+  className?: string;
+}
+
+export function Popover({
+  trigger,
+  children,
+  align = "center",
+  sideOffset = 4,
+  className,
+  ...props
+}: PopoverProps) {
+  return (
+    <ShadPopover {...props}>
+      <PopoverTrigger data-slot="popover-trigger" asChild>{trigger}</PopoverTrigger>
+      <PopoverContent align={align} sideOffset={sideOffset} 
+        data-slot="popover-content"
+        className={cn(
+          "data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2 z-99999 origin-(--radix-popover-content-transform-origin) rounded-md shadow-md outline-hidden min-w-full bg-neutral-background",
+          className,
+        )}>
+        {children}
+      </PopoverContent>
+    </ShadPopover>
+  );
+}
+
+export { PopoverTrigger, PopoverContent, PopoverAnchor }; 
```

Comment:
We will need an imperative way of rendering the Popover given it is, at the moment, mainly used in the DataManager ([show function](https://github.com/HumanSignal/label-studio-enterprise/blob/develop/web/apps/labelstudio/src/components/Popover/Popover.tsx#L140))

Reply:
wow, i wasn't aware that was used like that. ok no problem as much as I dislike usage like that :P